### ISSUE TITLE
[FEATURE] MobileNet V1 구현

### ISSUE THREAD

**Issue by jjj5306** [AnnyangService/HiMeow#3]
## 어떤 기능인가요?

기존의 코드 및 ResNet-50은 너무 무거워서 CPU 환경에서 학습시키기도 힘들고 모바일 환경에서 동작하기 좋지 않았다. 따라서 훨씬 가벼운 ModbileNet V1 아키텍쳐를 선택했고,
(link 2) 논문을 참고해보니 비슷한 주제에 대해 전이학습의 효과가 좋았다. 따라서 ImageNet을 바탕으로 학습한 모델의 파라미터를 동결시키고 전이학습하는 방향으로 구현한다.

## 작업 상세 내용

- [x] 데이터로더 구현
- [x] 모델 아키텍처 설정
- [x] Loss function 및 hyperparamerter 설정
- [x] 학습 loop 구현
- [x] 학습시키기

## 참고할만한 자료(선택)
[link1 , MobileNet V1 선택 근거](https://deview.kr/data/deview/2019/presentation/[115]%EC%96%B4%EB%94%94%EA%B9%8C%EC%A7%80+%EA%B9%8E%EC%95%84%EB%B4%A4%EB%8B%88_%EB%AA%A8%EB%B0%94%EC%9D%BC+%EC%84%9C%EB%B9%84%EC%8A%A4%EB%A5%BC+%EC%9C%84%ED%95%9C+%EA%B0%80%EB%B2%BC%EC%9A%B4+%EC%9D%B4%EB%AF%B8%EC%A7%80+%EC%9D%B8%EC%8B%9D_%EA%B2%80%EC%B6%9C+%EB%94%A5%EB%9F%AC%EB%8B%9D+%EB%AA%A8%EB%8D%B8.pdf)]

[link2, 전이학습 선택 근거](https://arxiv.org/pdf/2307.10501)
